enhancements/storage/csi-driver-install.md

```diff
@@ -79,61 +79,187 @@ along with in-tree driver and users could use both.
 
 ## Proposal
 
-We are currently considering using OLM for installation of CSI driver.
+We are currently considering using CVO and cluster-storage-operator (CSO) for installation of CSI driver.
+
+Each CSI driver is deployed via a dedicated operator (i.e. each CSI driver has its own), to accommodate differences
+between the CSI drivers. For example, Manila CSI driver operator can detect if Manila service is present in underlying
+OpenStack cloud and deploy the driver only when the service is present.
+
+* Each CSI driver will provide its own CRD, `<backend>Driver` in `csi.openshift.io` API group. E.g. `AWSEBSDriver` or
+  `ManilaDriver`. The CRD is non-namespaced - CSI drivers are global in the whole cluster.
+* Each CRD will allow only a single instance, named `cluster`.
+* Initially, the CRDs will be equal and provide only common
+  [`OperatorSpec`](https://github.com/openshift/api/blob/95abe2d2f4223d5931e418bf8e4d3773d16b42c0/operator/v1/types.go#L48)
+  and [`OperatorStatus`](https://github.com/openshift/api/blob/95abe2d2f4223d5931e418bf8e4d3773d16b42c0/operator/v1/types.go#L97).
+  In the future, each CSI driver may introduce its own configuration fields.
+* Each CSI driver runs in a dedicated namespace, typically `openshift-<backed>-csi-driver` (`openshift-aws-ebs-csi-driver`).
+  * TODO: or run everything in cluster-storage namespace? OLM installs into openshift-aws-ebs-csi-driver, we would need to delete it before starting a new driver in cluster-storage. 
+* TODO: where the operator runs? cluster-storage namespace?
```

Comment:
There are two open items for @openshift/storage team about namespaces where the operator / CSI driver will run.